package.json

```diff
@@ -40,6 +40,7 @@
     "prepare.dev": "EDGE_MODE=development ./scripts/prepare.sh",
```

Comment:
Get rid of `prepare.dev`, since the webpack script no longer cares about the flag.